### PR TITLE
doc(release): cherry pick process and PR template

### PR DESCRIPTION
**Pull request by Bobgy**
Part of https://github.com/kubeflow/pipelines/issues/3729

/assign @rmgogogo @jingzhang36

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -107,6 +107,17 @@ See the [Change Log](https://github.com/kubeflow/pipelines/blob/master/CHANGELOG
 
 ## Cherry picking PRs to release branch
 
+### Option - cherrypick-approved label
```
Comment (jingzhang36) [inline RELEASE.md:110]: Is this manual or auto?

Comment (Bobgy) [reply RELEASE.md:110]: which is manual or auto?
adding the label?

Comment (jingzhang36) [reply RELEASE.md:110]: Synced offline

**Comment by Bobgy** [comment]
@jingzhang36 so do you prefer to manually create cherry pick PRs by yourself? What about PRs you will approve?

**Comment by jingzhang36** [comment]
> @jingzhang36 so do you prefer to manually create cherry pick PRs by yourself? What about PRs you will approve?

I am ok either way. I added the label to a recent PR at https://github.com/kubeflow/pipelines/pull/4005.

**Comment by Bobgy** [comment]
OK, thanks!
/approve

**Pull request merged**